commit d3ea9550b5059e571516b94cbf9f3cd40933b2f7 (refs/changes/16/4416/3)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2018-06-22T19:35:29+00:00 (1 year, 4 months ago)
    
    MANTA-3783 waferlock should whitelist binder zones
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Approved by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/sapi_manifests/waferlock/template b/sapi_manifests/waferlock/template
index bff129b..3b61b52 100644
--- a/sapi_manifests/waferlock/template
+++ b/sapi_manifests/waferlock/template
@@ -24,6 +24,7 @@
 		"paths": ["/moray"],
 		"domains": [
 			"{{{moray_domain}}}",
+			"{{{BINDER_SERVICE}}}",
 			{{! Don't use pgstatsmon_domain, might not be set. }}
 			"pgstatsmon.{{{datacenter_name}}}.{{{dns_domain}}}"
 		],
@@ -38,7 +39,8 @@
 			"ops.{{{DOMAIN_NAME}}}",
 			"madtom.{{{DOMAIN_NAME}}}",
 			"pgstatsmon.{{{DOMAIN_NAME}}}",
-			"reshard.{{{DOMAIN_NAME}}}"
+			"reshard.{{{DOMAIN_NAME}}}",
+			"nameservice.{{{DOMAIN_NAME}}}"
 		],
 		"sapi_url": "{{{SAPI_URL}}}",
 		"sapi_services": ["manta/moray"],
